protostar/starknet_gateway/block_explorer/stark_scan_block_explorer.py

```diff
@@ -20,10 +20,8 @@ def get_name(self) -> str:
     def create_link_to_transaction(self, tx_hash: TransactionHash) -> Optional[str]:
         return f"{self._domain}/tx/0x{tx_hash:064x}"
 
-    def create_link_to_contract(
-        self, contract_address: ContractAddress
-    ) -> Optional[str]:
-        return f"{self._domain}/contract/0x{contract_address:064x}"
+    def create_link_to_contract(self, contract_address: Address) -> Optional[str]:
+        return f"{self._domain}/contract/0x{int(contract_address):064x}"
```

Comment:
we could just do regular interpolation here

Reply:
I wasn't sure about this one because two address representation and url params can change at different rate. At the other hand, these two concepts are both stable. I'll change it.